README.md

```diff
@@ -14,13 +14,13 @@ Official releases are tagged and can be found [here](https://github.com/GodotVR/
 The following branches are in active development:
 |  Branch   |  Description                |  Godot version  |
 |-----------|-----------------------------|-----------------|
-|  master   | Current development branch  |  Godot 3.4+     |
+|  master   | Current development branch  |  Godot 3.5+     |
 |  2.x      | Current stable release      |  Godot 3.4+     |
```

Comment:
I think this needs to change too now. `2.x` is already a historic release. 
Stable releases are tagged, not managed by branches.

Reply:
I'll just remove the 2.x branch from the list as it's describing the active development branches and I doubt we'll revisit 2.x